bodhi/server/validators.py

```diff
@@ -1323,6 +1323,9 @@ def validate_from_tag(request: pyramid.request.Request, **kwargs: dict):
         koji_client = buildsys.get_session()
         taginfo = koji_client.getTag(koji_tag)
 
+        if not taginfo['extra'].get('sidetag'):
+            request.errors.add('body', 'from_tag', "The supplied tag doesn't contain a side tag")
+
```

Comment:
The `extra` dict is always in `taginfo`, the KojiHub code takes care of that. For a side tag, it would e.g. look like this (see https://koji.stg.fedoraproject.org/koji/taginfo?tagID=7715 or its corresponding `koji_client.getTag(...)` API output):

```
'extra': {'sidetag_user': 'nphilipp', 'sidetag': True, 'sidetag_user_id': 11},
```

For a normal tag, the `extras` dict would be missing the `sidetag` field, i.e. `taginfo['extra'].get('sidetag')` returns `False`.